app/views/research_sessions/preview.html.erb

```diff
@@ -157,18 +136,112 @@
           <% end %>
         </p>
       <% end %>
-    </section>
-  <% end %>
-  <% if @research_session.incentives_enabled %>
-    <section>
-      <h3 class="subtitle-small">Incentives</h3>
-      <p>
-        We are offering
-        <%= edit_link_for(:incentive_value) do %><%= @research_session.incentive_text %><% end %>
-        for participation in this session.
-      </p>
-    </section>
-  <% end %>
+    <% end %>
+  </section>
+
+  <section>
+    <h3 class="subtitle-small" id="confidential">
+      Will anyone know what <%= i_or_they %> say in the discussion?
+    </h3>
+
+    <p>
+      The only people who will hear what <%= you_or_your_child %> <%= say_or_says %> in the session
+      will be the researchers running the session, other young people taking part, and
+      service workers facilitating the session.
+    </p>
+
+    <p>
+      <%= edit_link_for(:shared_with) do %>
+        <%= shared_with_lookup(@research_session.shared_with) %>
+      <% end %>
+    </p>
+
+    <p>
+      Barnardo’s will hold research data for <%= edit_link_for(:shared_duration) %>
+      after the closure of this project, after which it will be deleted.
+      Personal data is stored in a safe and secure way.
+    </p>
+    <p>
+      You can contact Jason Caplin to ask us to delete your personal data at any time.
+    </p>
+  </section>
+
+  <section>
+    <h3 class="subtitle-small" id="sharing-information">Sharing information</h3>
+    <p>
+      We may have to share information about <%= you_or_your_child %> without
+      your consent if we think <%= you_or_they %> or someone else is at risk
+      of serious harm. Where possible, the lead researcher speak to you first
+      and ask you what you want to do about the situation.
+    </p>
+  </section>
+
+  <section>
+    <h3 class="subtitle-small" id="about-taking-part">About taking part</h3>
+    <p>
+      <%= you_or_your_child.capitalize %> <%= @research_session.unable_to_consent? ? 'does' : 'do' %>
+      not have to take part in this session. It is your decision<%= ' as the parent/carer' if @research_session.unable_to_consent? %>.
+    </p>
+    <p>
+      We will also ask <%= you_or_your_child %> if <%= you_or_they %> want to take
+      part before the session.
+    </p>
+    <p>
+      Don’t worry! If you do not want <%= 'the child in your care' if @research_session.unable_to_consent? %>
+      to take part it will not affect the service <%= you_or_they %> receive.
+    </p>
+    <p>
+      <%= you_or_they.capitalize %> can change your mind at any point for any
```

Comment:
Currently it comes out as

>  They can change your mind at any point for any reason - just let us know!

Please change lines 194 and 195 to:

> *Your child / the child in your care* can change *their* mind during the session at any point for any reason by letting the researcher know *they* want to stop.
AND
> *You* can change *your* mind during the session at any point for any reason by letting the researcher know *you* want to stop.

Reply:
I love that top error. Can we keep it?